2024-04-04 12:17:34 INFO [32m pytest tests/backend/test_create_project.py::TestProjectCreateWithInvalidData::test_create_project_with_invalid_ids[Roles.PROJECT_ADMIN-$651DrOGIOw] (setup)[0m
curl -X GET 'http://localhost:8111/authenticationTest.html?csrf' \
-H 'User-Agent: python-requests/2.31.0'\
-H 'Accept-Encoding: gzip, deflate'\
-H 'Accept: */*'\
-H 'Connection: keep-alive'\
-H 'Authorization: Basic OjU5NTQwNTE3MTQ2NjUyNTE0MDM=' \
-d ''
2024-04-04 12:17:34 INFO [32m pytest tests/backend/test_create_project.py::TestProjectCreateWithInvalidData::test_create_project_with_invalid_ids[Roles.PROJECT_ADMIN-$651DrOGIOw][0m
curl -X POST 'http://localhost:8111/app/rest/projects' \
-H 'User-Agent: python-requests/2.31.0'\
-H 'Accept-Encoding: gzip, deflate'\
-H 'Accept: application/json'\
-H 'Connection: keep-alive'\
-H 'Content-Type: application/json'\
-H 'X-TC-CSRF-Token: 33e59a9f-04c1-4f67-bd69-05820ac8b3ba'\
-H 'Cookie: RecentLogin=; TCSESSIONID=B9A9750CE36D5358839E8F814356B986'\
-H 'Content-Length: 111'\
-H 'Authorization: Basic OjU5NTQwNTE3MTQ2NjUyNTE0MDM=' \
-d '{"parentProject": {"locator": "_Root"}, "name": "true", "id": "$651DrOGIOw", "copeAllAssociatedSettings": true}' 

2024-04-04 12:17:34 INFO 	RESPONSE:
STATUS_CODE: [31m500[0m
DATA: [31mError has occurred during request processing, status code: 500 (Internal Server Error).
Details: jetbrains.buildServer.serverSide.InvalidIdentifierException: Project ID "$651DrOGIOw" is invalid: starts with non-letter character '$'. ID should start with a latin letter and contain only latin letters, digits and underscores (at most 225 characters).
Error occurred while processing this request.[0m